package.json

```diff
@@ -22,5 +26,31 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jest": "^21.25.0",
     "regenerator-runtime": "^0.12.1"
+  },
+  "directories": {
+    "test": "tests"
+  },
+  "bugs": {
+    "url": "https://github.com/Pocket/apollo-cursor-pagination/issues"
+  },
+  "homepage": "https://github.com/Pocket/apollo-cursor-pagination#readme",
+  "publishConfig": {
+    "cache": "~/.npm",
+    "access": "public"
+  },
+  "release": {
+    "branches": [
+      "main"
+    ],
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/npm",
+      "@semantic-release/github"
```

Comment:
need to add https://github.com/Pocket/terraform-modules/blob/main/package.json#L77-L82 so that the package version updates automatically